website/style.css

```diff
@@ -154,7 +154,13 @@ a {
     content: "\00a0";
 }
 
-code, pre {
+.module {
```

Comment:
No.

Reply:
Okay, but the lack of such a mechanism means that placing the module name at the front of the title severely reduces the readability of the generated index. I agree that using this as in this example is suboptimal:

```
title = "<span class=module>builtin:</span> Builtin Functions and Variables"
```

Ideally there would be an alternative to the `title` attribute that allows specifying a module name as a prefix to the title text and that is rendered in a distinct, fixed with, manner. If I were using HTML or Markdown directly to write the index document I would use a table.